zlib/zlib.cabal

```diff
@@ -99,15 +100,21 @@ library
   if flag(non-blocking-ffi)
     cpp-options:   -DNON_BLOCKING_FFI
 
-  if flag(bundled-c-zlib) || impl(ghcjs) || os(ghcjs) || arch(wasm32)
-    build-depends: zlib-clib
+  -- Cross-platform builds (such as JS and WASM) must have access
+  -- to C sources, so using zlib-clib unconditionally.
+  --
+  -- On Windows zlib is shipped with GHC starting from 7.10
+  -- at mingw/lib/libz.a. Yet linking against it proves unreliable:
+  -- while GHC always links against mingw/lib whatever the PATH,
```

Comment:
This is a bit misleading, since GHC won't consult your `PATH` at link time, only at runtime. It might be better to phrase this as:

> On Windows, zlib is shipped as part of GHC's mingw/lib directory, which GHC always includes in its linker search path. However, there is no guarantee that zlib1.dll (the corresponding shared library) will be available on the user's PATH at runtime, making it risky to depend upon (see #65 for what can go wrong). Thus, we resort to zlib-clib unless pkg-config is available.

Or something like that.

Reply:
Thanks! I'll use the suggested wording.